assert.go

```diff
@@ -27,20 +27,28 @@ func SetConfig(config Config) {
 //
 // WARN: This assertion is live!
 func Assert(condition bool, msg string, values ...any) {
-	assert(condition, msg, values...)
+	assert(condition, msg, 2, values...)
 }
 
 // Assert panics if the condition is false. Configurable via SetConfig.
-func assert(condition bool, msg string, values ...any) {
+// skipFrames is the number of stack frames to skip when getting the source context. 
+// By default, we skip 2 frames:
+//  1. The assert() function itself
+//  2. The Assert()/Debug() function that called assert()
+func assert(condition bool, msg string, skipFrames int, values ...any) {
```

Comment:
This part of the comment should not be here IMO. 

```suggestion
// skipFrames is the number of stack frames to skip when getting the source context. 
func assert(condition bool, msg string, skipFrames int, values ...any) {
```

Technically `assert` does not know it gets called by `Assert()`/`Debug()`. This comment should be moved to where we pass the `2`, explaining why we're passing a `2`. We technically also could now turn `skipFrames` into a constant since `2` gets used multiple times, but I think I'd rather keep it local to the `Assert()` and `Debug()` functions respectively, so that they "own" the number of frames to be skipped in their respective cases.